Tests/MongoSwiftSyncTests/UnifiedTestRunner/ExpectedEventsForClient.swift

```diff
@@ -18,8 +18,13 @@ struct ExpectedEventsForClient: Decodable {
     /// (excluding ignored events).
     let events: [ExpectedEvent]
 
+    /// Specifies how the `events` array is matched against observed events.  If false, observed events after all
+    /// specified events have matched MUST cause a test failure; if true, observed events after all specified events
+    /// have been matched MUST NOT cause a test failure. Defaults to false.
+    let ignoreExtraEvents: Bool?
```

Comment:
since we already fill in a default value of `false` for this in the Decodable implementation below, I think we could just make this non-optional here, which would save us from having to do `?? false` when we pass this value into `matchesEvents`.

Reply:
Ohh okay yeah that makes sense. got confused when you said "save us having to do `??` " but that was for the `testFiles` function and I thought u meant for the `init` . Made the edits, hopefully it compiles this time